### PR TITLE
Move session start time to the NWBExtractor

### PR DESCRIPTION
**Pull request by h-mayorquin**
Changed the methods to extract the session start time from the extractor to the interface where it belongs.

### CODE REVIEW

File: src/clandinin_lab_to_nwb/brezovec/brezovecimagingextractor.py [hunk 1]
```diff
@@ -247,15 +204,12 @@ def get_plane_acquisition_rate(self, cycle) -> float:
...
-            float(frame.attrib["absoluteTime"]) for frame in frame_elements
-        ]  # TODO should we use absoluteTime or relativeTime?
-        timestamps = [absolute_times[t] for t in np.arange(0, len(absolute_times), self._num_planes)]
-        return np.array(timestamps)
-
-    def get_series_datetime(self):
-        series_datetime = read_session_date_from_file(folder_path=self.folder_path)
-        return series_datetime
+        relative_times = [
+            float(frame.attrib["relativeTime"]) for frame in frame_elements
```
Comment (h-mayorquin) [inline src/clandinin_lab_to_nwb/brezovec/brezovecimagingextractor.py:208]: Changing this to relativeTime as in the comment.

File: src/clandinin_lab_to_nwb/brezovec/brezovecimaginginterface.py [hunk 2]
```diff
@@ -79,28 +82,29 @@ def get_metadata(self) -> DeepDict:
             imaging_plane=imaging_plane_name,
             scan_line_rate=1 / float(xml_metadata["scanLinePeriod"]),
             rate=self.imaging_extractor.get_sampling_frequency(),
-            description=f"{self.imaging_purpose} imaging data ({indicator}) acquired from the Bruker Two-Photon Microscope",
-            unit="px",
+            description=f"{self.imaging_purpose} imaging data ({indicator})",
+            unit="voxels",
```
Comment (h-mayorquin) [inline src/clandinin_lab_to_nwb/brezovec/brezovecimaginginterface.py:86]: I am not sure about this @alessandratrapani what do you think?

Comment (alessandratrapani) [reply src/clandinin_lab_to_nwb/brezovec/brezovecimaginginterface.py:86]: Actually it should be n.a. as in the default value. My mistake, I misunderstood the meaning here. For reference see this [discussion](https://github.com/NeurodataWithoutBorders/nwb-schema/issues/460#:~:text=units%20of%20collected%20data%20%2D%20virtually%20no%20one%20knows%20it.%20At%20best%20you%20can%20express%20it%20in%20photon%20numbers%20(unitless)%20or%20W/pixel%20but%20the%20calibration%20is%20tricky%20and%20a%20very%20few%20labs%20could%20provide%20this%20information.)

**Review by alessandratrapani — APPROVED** [review]
Look good to me!

File: src/clandinin_lab_to_nwb/brezovec/brezovecimaginginterface.py (hunk 2 above)
Comment (h-mayorquin) [reply src/clandinin_lab_to_nwb/brezovec/brezovecimaginginterface.py]: OK, changed, thanks!

**Pull request merged**